assembl/static2/js/app/utils/linkify.js

```diff
@@ -25,8 +25,10 @@ export function addProtocol(url: string): string {
   return hasProtocol ? url : `https://${url}`;
 }
 
-export function addIframeForMindManager(html: string): string {
-  const url = /(<a href="(https:\/\/share.mindmanager.com[^\s]+)".*<\/a>)/gi;
+export function addIframeForMindMapping(html: string): string {
+  const url = html.includes('mindmanager')
```

Comment:
This is a good first effort. But it's exclusively binary. What if tomorrow there's another mapping tool they want to embed. You'll have to change this logic completely. I suggest you build an array of regex's, then do a reduce on which mapping fits the given html. And finally, apply the mapping the works.